definitions/private-law/json/AuthorisationCaseEvent/ConfidentialityCheck/AuthorisationCaseEvent-preview.json

```diff
@@ -6,18 +7,20 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "hearing-centre-team-leader"
+          "hearing-centre-team-leader",
+          "caseworker-privatelaw-courtadmin",
```

Comment:
are you sure this gets picked up only in preview?

Reply:
yes. generate-excel-preview is decorated like that only